.github/CODEOWNERS

```diff
@@ -66,7 +66,7 @@
 
 /packages/fx-core/src/common/deps-checker @qinezh @a1exwang @kimizhu @swatDong
 /packages/fx-core/tests/common/deps-checker @qinezh @a1exwang @kimizhu @swatDong
- 
+
```

Comment:
There's an extra space in this line. The tool used to detect files with no codeowner can't handle this line correctly, so remove this extra space.